backend/compact-connect/lambdas/custom-resources/tests/function/test_handlers/test_compact_configuration_uploader.py

```diff
@@ -0,0 +1,179 @@
+import json
+from decimal import Decimal
+from datetime import date
+
+from tests.function import TstFunction
+from moto import mock_aws
+from boto3.dynamodb.conditions import Key
+
+TEST_ENVIRONMENT_NAME = "test"
+
+
+def generate_single_root_compact_config(compact_name: str, active_environments: list):
+    return {
+        "compactName": compact_name,
+        "compactCommissionFee": {
+            "feeType": "FLAT_RATE",
+            "feeAmount": 3.5
+        },
+        "compactOperationsTeamEmails": [],
+        "compactAdverseActionsNotificationEmails": [],
+        "compactSummaryReportNotificationEmails": [],
+        "activeEnvironments": active_environments
+    }
+
+
+def generate_single_jurisdiction_config(jurisdiction_name: str, postal_abbreviation: str, active_environments: list):
+    return {
+        "jurisdictionName": jurisdiction_name,
+        "postalAbbreviation": postal_abbreviation,
+        "jurisdictionFee": 100,
+        "militaryDiscount": {
+            "active": True,
+            "discountType": "FLAT_RATE",
+            "discountAmount": 10
+        },
+        "jurisdictionOperationsTeamEmails": [],
+        "jurisdictionAdverseActionsNotificationEmails": [],
+        "jurisdictionSummaryReportNotificationEmails": [],
+        "jurisprudenceRequirements": {
+            "required": True
+        },
+        "activeEnvironments": active_environments
+    }
+
+
+def generate_date_string():
+    # yes, there is always a chance that the tests are run precisely at midnight
+    # but that's a risk we're willing to take
```

Comment:
implication being? a comparison would fail since this function might return different dates during the same run of a test?

Reply:
Yes, I've updated the comment to make this more explicit